Jenkinsfile

```diff
@@ -108,231 +114,388 @@ pipeline {
-                    sh "./scripts/repackage_bundle.sh *.tgz"
-                    sh "mv zowe-cli-package.zip zowe-cli-plugins-${ZOWE_CLI_BUNDLE_VERSION}.zip"
-
-                    archiveArtifacts artifacts: "zowe-cli-plugins-${ZOWE_CLI_BUNDLE_VERSION}.zip"
-
-                    // Remove all tgzs after bundle is archived
-                    sh "rm -f *.tgz"
-                }
-            }
-        }
-        /************************************************************************
-         * STAGE
-         * -----
-         * Build Zowe NodeJS SDK Bundle
-         *
-         * TIMEOUT
-         * -------
-         * 10 Minutes
-         *
-         * EXECUTION CONDITIONS
-         * --------------------
-         * - Always
-         *
-         * DECRIPTION
-         * ----------
-         * Gets the latest version of the Zowe NodeJS SDK package from NPM
-         * Creates an archive with 'fat' versions of the Plugins -
-         *   dependencies are bundled.
-         *
-         * OUTPUTS
-         * -------
-         * A Zowe NodeJS SDK Archive.
-         ************************************************************************/
-        stage('Create Zowe NodeJS SDK Bundle') {
-            when {
-                allOf {
-                    expression {
-                        return BRANCH_NAME.equals(MASTER_BRANCH)
+            failFast true
+            parallel {
+                stage('Create Bundles (LTS)') {
+                    stages {
+                        /************************************************************************
+                        * STAGE
+                        * -----
+                        * Build Zowe CLI Bundle
+                        *
+                        * TIMEOUT
+                        * -------
+                        * 10 Minutes
+                        *
+                        * EXECUTION CONDITIONS
+                        * --------------------
+                        * - Always
+                        *
+                        * DESCRIPTION
+                        * ----------
+                        * Gets the latest version of the Zowe CLI and SCS plugin from Zowe
+                        * Artifactory. Creates an archive with 'fat' versions of the CLI and Plugin -
+                        *  dependencies are bundled.
+                        *
+                        * OUTPUTS
+                        * -------
+                        * A Zowe CLI Archive containing Zowe CLI and Zowe CLI Secure Credential Store Plugin
+                        ************************************************************************/
+                        stage('CLI Core (LTS)') {
+                            steps {
+                                timeout(time: 10, unit: 'MINUTES') {
+                                    dir("lts") {
+                                        sh "mkdir -p licenses && cd licenses && cp ../../zowe_licenses_full.zip zowe_licenses_full.zip"

```

Comment:
Curious if the license.zip file will be included twice in the Bundle.zip.
This same comment may apply to other places. 😋

Reply:
Not sure I understand this comment 😢 Each stage is creating a separate .zip file, so I don't think the license.zip file should be getting duplicated anywhere.

Reply:
I thought that the resulting zip file (with the licenses folder scs and zowe-cli tgz) was going to also include the license.zip
But after seeing the updated build, that's not the case 😋